<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
 <testsuite name="Salesforce Development_script" package="DavesSupportSuite.suite" timestamp="2025-02-28 13:45:08.219 -0500"
 tests="1"  failures="1" errors="0" time="106.612000">
  <properties>
   <property name="RunDate" value="2025-02-28 13:45:08.219 -0500"/>
   <property name="Status" value="Aborted"/>
  </properties>
  <testcase name="Salesforce Development.script" assertions="3" status="failure"  time="106.603">
    <failure type="LogError" message="Authentication challenge received, but no credentials have been provided." />
  </testcase>
 </testsuite>
</testsuites>
